### DISCUSSION TITLE
Feature: Add Translations to Cloudlog

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#632] [Ideas / Feature Requests]
With Cloudlogs users being international getting lots of comments about making it translatable which is do able using https://codeigniter.com/userguide3/libraries/language.html just requires building files and changing the views which does add work but long term probably worthwhile.

I'm guessing this is of interest to everyone and who would like to get involved

**Comment by phl0** [comment]
I like the idea. Could provide German translations. 😎

**Comment by magicbug** [comment]
Guess best idea is to start per section of Cloudlog, this is likely to take a while

**Comment by magicbug** [comment]
Decided to have a play and do the front dashboard page its now committed and you can see the language file at https://github.com/magicbug/Cloudlog/blob/master/application/language/english/general_words_lang.php

**Comment by cyber-atomus** [comment]
Ha, posted on tweetear an idea and it's already alive! Dang that was fast! Reading abt codeigniter took me a while, but I did almost the same job as @magicbug. It's there: https://github.com/cyber-atomus/Cloudlog - also translated some stuff to Polish already...

**Comment by magicbug** [comment]
Works for myself, PRs are accepted, I only did it roughly to see how it could work 👍🏻

**Comment by phl0** [comment]
I did add basics for German with PR https://github.com/magicbug/Cloudlog/pull/720
